industry_benchmarks/utils/data_gathering.py

```diff
@@ -453,14 +474,257 @@ def gather_ligand_scores(
+
+    Returns
+    -------
+        failed_results: list[str]
+        A list of edge names which have missing results
+    """
+    # workout the expected number of results
+    # assuming 3 repeats of each solvent and complex transformation
+    missing_results = []
+    all_results = {}
+    for edge in alchemical_network.edges:
+        all_results[edge.name] = []
+    expected_results = len(all_results) * 3
+
+    # map the transformation to the results files
+    for results_folder in results_folders:
+        for results_file in tqdm.tqdm(results_folder.glob("*.json"), desc=f"Processing results in {results_folder}", ncols=80):
+            # run checks on the end results
+            result = load_results_file(file_name=results_file)
+            if result is not None:
+                # find the cleaned up results file
+                simulation_data_file = find_data_folder(result=result)
+                # work out the name of the transform and add it to the dict
+                transformation_name = get_transform_name(result=result, alchemical_network=alchemical_network)
+                # collect the paths to the results files and the structural
+                if transformation_name in all_results and simulation_data_file is not None:
+                    all_results[transformation_name].append((results_file, simulation_data_file))
+                elif transformation_name not in all_results:
+                    error_message = (f"Found a result for {transformation_name} which was not expected "
+                                     f"from the alchemical network.")
+                    raise ValueError(error_message)
+
+    # Write stats on the number of transformations found
+    found_results = sum([len(v) for v in all_results.values()])
+    print(f"Total results found {found_results}/{expected_results} indicating {expected_results - found_results} failed transformations.")
+    # check we have a connected network
+    if not check_network_is_connected(results_data=all_results, alchemical_network=alchemical_network):
+        raise ValueError(f"The network built from the complete results is disconnected, some simulations may still be"
+                         f"running or needed restarting. Reproducible edge failures may require extract edges which "
+                         f"can be generated using the `fix_networks.py` script.")
+
+    # move the results to the output folder
+    for transformation_name, results in tqdm.tqdm(all_results.items(), desc="Collecting files", total=len(all_results), ncols=80):
+        for i, result_file, results_dir in enumerate(results):
+            output_path = output_dir.joinpath(transformation_name, f"repeat_{i}")
+            output_path.mkdir(parents=True, exist_ok=False)
+            result_file: pathlib.Path
+            shutil.copy(result_file, output_path.joinpath(result_file.name))
```

Comment:
@IAlibay @hannahbaumann as the chemical system components are present in the transformation result should we not be copying these files?

Reply:
Sorry I'm probably being silly, which files do you mean?